models/gitm/model_mod.f90

```diff
@@ -2101,19 +2101,17 @@ subroutine unpack_data0d(data0d, ivar, ncid, define)
       call error_handler(E_MSG,routine,string1,source,revision,revdate)
    end if
 
-   call nc_define_double_variable(ncid, gitmvar(ivar)%varname, (/ LON_DIM_NAME, LAT_DIM_NAME, ALT_DIM_NAME /) )
```

Comment:
I'm missing why this line is being removed.  Is this related to the AIRS converter?

Reply:
The declaration of the scalar variable as a 3D variable is simple misleading. The thing is read as a scalar from a namelist, so it will never be 3D ...